doc/Language/glossary.pod6

```diff
@@ -838,6 +838,11 @@ X<|Perl>
 
 The Perl programming language in its many forms.
 
+X<|Perl 6>
+=head1 Perl 6
```

Comment:
There's a quirk, though... We always use non-breaking space between Perl and 6. It looks exactly the same from here, but if you didn't use that, it will not pass tests.